docs/logging-usage.rst

```diff
@@ -0,0 +1,307 @@
+Using the API
+=============
+
+
+Authentication and Configuration
+--------------------------------
+
+- For an overview of authentication in ``gcloud-python``,
+  see :doc:`gcloud-auth`.
+
+- In addition to any authentication configuration, you should also set the
+  :envvar:`GCLOUD_PROJECT` environment variable for the project you'd like
+  to interact with. If you are Google App Engine or Google Compute Engine
+  this will be detected automatically.
+
+- After configuring your environment, create a
+  :class:`Client <gcloud.logging.client.Client>`
+
+  .. doctest::
+
+     >>> from gcloud import logging
+     >>> client = logging.Client()
+
+  or pass in ``credentials`` and ``project`` explicitly
+
+  .. doctest::
+
+     >>> from gcloud import logging
+     >>> client = logging.Client(project='my-project', credentials=creds)
+
+
+Writing log entries
+-------------------
+
+Write a simple text entry to a logger.
+
+.. doctest::
+
+   >>> from gcloud import logging
+   >>> client = logging.Client()
+   >>> logger = client.logger('log_name')
+   >>> logger.log_text("A simple entry")  # API call
```

Comment:
Is there a way to automagically figure this out? So that I could write:

```python
logger.log('Some text!')
logger.log(message='this is some data', severity='ERROR')
logger.log(exception)
```

?